test/c/comment-strip.c

```diff
@@ -107,3 +107,27 @@ int two_liner(int foo, int bar);
  	 */
 int two_liner_whitespace(int foo, int bar);
 
+/**/
+struct empty_comment_0 {
+	/** The embedding struct is not documented. */
+	int description;
+};
+
+/***/
+struct empty_comment_1 {
+	/** The canonical empty documentation comment. */
+	int description;
+};
+
+/** */
+struct empty_comment_2 {
+	/** Another empty documentation comment. */
+	int description;
+};
+
+/**
+ */
+struct empty_comment_3 {
+	/** Another empty documentation comment. */
+	int description;
+};
```

Comment:
What about top level comments? Any chance it might be an issue for `autosection`? (I plead ignorance by the way.)

Reply:
Did not try yet. But seems like it could be a **feature**! You could use it to end a section (the future version with `:members:` support).

Reply:
Ahahah, maybe...

I'm expecting it to work fine right now, but I'd welcome a unit test for selecting sections before and after one of these at least. In case we inadvertently break it later on.

Reply:
Dropped it completely. Can be added later if needed.

Reply:
Yeah, feels complete now. Thanks.